glib/src/value.rs

```diff
@@ -246,6 +246,14 @@ where
 }
 
 /// Trait to convert a value to a `Value`.
+///
+/// Similar to other common conversion traits, the following invariants are guaranteed:
+/// 
+/// - **Invertibility**: `x.to_value().get().unwrap() == x`. In words, `FromValue` is the inverse of `ToValue`.
+/// - **Idempotence**: `x.to_value() == x.to_value().to_value()`.
+///   In words, applying `ToValue` multiple times yields the same result as applying it once.
+///   Idempotence also applies the other way around: `value.get::<Value>()` is a no-op.
+
```

Comment:
Maybe add a comment here about `BoxedValue` for the rare case to actually store a `Value` inside a `Value`

Reply:
Please remove this empty line.